Vagrantfile

```diff
@@ -3,7 +3,7 @@
 
 require 'yaml'
 
-settings = YAML.load_file('settings.yml')
+settings = File.exist?('settings.yml') ? YAML.load_file('settings.yml') : {}
```

Comment:
I have mixed feelings here. On one hand we don't require the settings.yml file to exist since we have default values for most of the options, but on the other hand we users usually don't read the documentation and will not know what will be the cluster configuration that vagrant will create unless we force them to edit the settings.yml.

IMO I think we should raise an error in the case there is no settings.yml file